terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/test/java/org/terasoluna/gfw/web/mvc/support/CompositeRequestDataValueProcessorTest.java

```diff
@@ -51,14 +52,12 @@ public void setup() {
     @Test
     public void testProcessActionSameActionAndResult01() {
         // Set mock behavior
-        // for Spring3
         when(requestDataValueProcessor.processAction(
                 (HttpServletRequest) (anyObject()), anyString())).thenReturn(
                         "action");
-        // for Spring4
         when(requestDataValueProcessor.processAction(
-                (HttpServletRequest) (anyObject()), anyString(), anyString()))
-                        .thenReturn("action");
+                (HttpServletRequest) (anyObject()), anyString(), Mockito
+                        .isNull())).thenReturn("action");
```

Comment:
- [ ] It's better `ArgumentMatchers#nullable` than `Mockito#isNull`. And please use static import.

Reply:
Changed Mockito#isNull to ArgumentMatchers#nullable.

Reply:
OK